.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+services:
+  - xvfb
+  - mysql
+python:
+    - "3.5"
+before_install:
+    - "export DISPLAY=:99"
+    - wget https://github.com/mozilla/geckodriver/releases/download/v0.25.0/geckodriver-v0.25.0-linux64.tar.gz
+    - mkdir geckodriver
+    - tar -xzf geckodriver-v0.25.0-linux64.tar.gz -C geckodriver
+    - export PATH=$PATH:$PWD/geckodriver
+    - export BOKCHOY_HEADLESS=true
+    - mysql -e 'set global wait_timeout=3600'
+    -  mkdir -p var  # for var/workbench.log
+install:
+    - "pip install -r requirements.txt"
+    - "pip install -r test_requirements.txt"
+    - "pip install mysqlclient==1.4.6"
```

Comment:
since we are not running CI tests with MySQL DB, we can remove this.

Reply:
@shafqatfarhan Its done.